src/core/utils/spec-preprocess.ts

```diff
@@ -258,6 +256,10 @@ export function traverseToFixSpecDownstream(spec: GoslingSpec | SingleView, pare
             // Override styles
             track.style = getStyleOverridden(spec.style, track.style);
             if (IsOverlaidTrack(track)) {
+                // Remove the dummy tracks from an overlay track
+                track.overlay = track.overlay.filter(overlayTrack => {
+                    return !('type' in overlayTrack && overlayTrack.type == 'dummy-track');
+                });
```

Comment:
We don't allow dummy tracks to be overlaid on other tracks

Reply:
It looks like a dummy track in a circular view hide a regular track, e.g., 

```ts
{
      "layout":  "circular",
      "tracks": [
        {
          "type": "dummy-track",
          "title": "here",
          "style": { "background": "grey" },
          "width": 100
        },
        { ... } // ← regular track that is hidden in a rendered visualization
   ]
}
```

We can simply remove the dummy track in a circular view, similar to what you did for overlaid tracks, somewhere in this function.

Reply:
Done! Dummy track in circular layout now removed